testcase/tachyonfake/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:22-slim
+
+# Install curl for health check and tini for process management
+RUN apt-get update && apt-get install -y curl tini && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+# Copy package files
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the tachyon fake server implementation
+COPY src/tachyonServer.fake.ts ./src/
+COPY src/tachyonTypes.ts ./src/
+COPY src/tachyonTypes.test.ts ./src/
+COPY src/oauth2Client.ts ./src/
+COPY src/oauth2Client.test.ts ./src/
+COPY tsconfig.json ./
```

Comment:
Short comment: I would advise against doing this, and also creating a dedicated tachyon fake package: it just makes it harder to maintain, now in this file you manually encoded dependency structure, and in the new package.json, there are deps, which need to be keep in sync with the root one.